src/components/data-grid/types.ts

```diff
@@ -329,10 +331,45 @@ export interface DataGridIconProps {
 }
 
 export interface DataGridActionsProps {
-  api: GridApi;
+  gridApi: GridApi;
+  gridColumnApi: ColumnApi;
   columns: DataGridColumn[];
   rowsSelected: number;
   translations: Translations;
   onBulkDelete?: (ids: string[]) => Promise<void>;
   hideDownload?: boolean;
 }
+
+export interface PrintParams {
+  PDF_HEADER_COLOR?: string;
+  PDF_INNER_BORDER_COLOR?: string;
+  PDF_OUTER_BORDER_COLOR?: string;
+  PDF_ODD_BKG_COLOR?: string;
+  PDF_EVEN_BKG_COLOR?: string;
+  PDF_HEADER_HEIGHT?: number;
+  PDF_ROW_HEIGHT?: number;
+  PDF_PAGE_ORIENTATION?: PageOrientation;
+  PDF_WITH_CELL_FORMATTING?: boolean;
+  PDF_WITH_COLUMNS_AS_LINKS?: boolean;
+  PDF_SELECTED_ROWS_ONLY?: boolean;
+  PDF_WITH_HEADER_IMAGE?: boolean;
+  PDF_WITH_FOOTER_PAGE_COUNT?: boolean;
+  PDF_LOGO?: string;
+}
+
+interface PdfCell {
+  text?: string;
+  style?: string;
+  link?: string;
+  color?: string;
+  decoration?: string;
+  [key: string]: any;
```

Comment:
This indexer renders the idea of interface a bit useless. I'd remove this. If there are code parts which start to complain, then I'd resolve them either by:
- extending PdfCell
- using | operator to apply another type